test/extensions/filters/http/ext_proc/streaming_integration_test.cc

```diff
@@ -69,6 +69,12 @@ class StreamingIntegrationTest : public HttpIntegrationTest,
       const auto addr = Network::Test::getCanonicalLoopbackAddress(ipVersion());
       const auto addr_port = Network::Utility::getAddressWithPort(*addr, test_processor_.port());
       setGrpcService(*proto_config_.mutable_grpc_service(), "ext_proc_server", addr_port);
+      // Insert some extra metadata. This ensures that we are actually passing the
+      // "stream info" from the original HTTP request all the way down to the
+      // ext_proc stream.
+      auto* metadata = proto_config_.mutable_grpc_service()->mutable_initial_metadata()->Add();
```

Comment:
How does the test verify that the metadata gets passed down?

Reply:
I was focused on ensuring that it doesn't crash, and not focusing on whether the underlying Envoy gRPC library sends the metadata to the server. But I added some test code to verify that as well. Thanks!